src/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -565,6 +566,8 @@ public static void VerifyWithRevocation()
 
                             Console.WriteLine(
                                 $"{nameof(VerifyWithRevocation)}: online attempt {i} - errors at depth {j}: {allFlags}");
+
+                            Thread.Sleep(1000);
```

Comment:
This sleeps one second per cert in the chain.  Right after inner for loop is best :smile:.  (currently line 571/574).

Once it's inside the outer loop / outside the inner loop I'm good with the change.